packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.jsx

```diff
@@ -72,6 +72,9 @@ export const IdvDocSubmitOnSignup = ({ citizen_data, has_previous, onPrevious, o
     const validateFields = values => {
         const errors = {};
         const { document_type, document_number } = values;
+        const sequential_number = isSequentialNumber(document_number);
+        const recurring_number = recurringNumberRegex(document_number);
```

Comment:
can we change the name of this  varible and function name to something like this
```suggestion
        const has_recurring_number = isRecurringNumberRegex(document_number);
```